### PR TITLE
feat(voting): Add Registry, AddressWhitelist, and Store to Voting subgraph and add Polygon template

### PR DESCRIPTION
**Pull request by nicholaspai**

### CODE REVIEW

File: packages/voting/README.md [hunk 1]
```diff
@@ -4,3 +4,15 @@
 
 - Mainnet: `yarn prepare:mainnet && yarn deploy:mainnet`
 - Kovan: `yarn prepare:kovan && yarn deploy:kovan`
+- Polygon: `yarn prepare:polygon && yarn deploy:polygon`
+
+## Caveats
```
Comment (chrismaree) [inline packages/voting/README.md:9]: 👍

File: packages/voting/manifest/data/kovan.json [hunk 2]
```diff
@@ -1,9 +1,12 @@
 {
   "network": "kovan",
-  "votingTokenAddress": "0x489Bf230d4Ab5c2083556E394a28276C22c3B580",
```
Comment (chrismaree) [inline packages/voting/manifest/data/kovan.json:3]: why remove this?

Comment (chrismaree) [reply packages/voting/manifest/data/kovan.json:3]: nvm I guess it's in the `token` package

Comment (nicholaspai) [reply packages/voting/manifest/data/kovan.json:3]: We don't use it anywhere, its hardcoded in `utils/constants`

**Pull request merged**